test/integration/targets/redshift/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+posix/ci/cloud/group4/aws
+shippable/aws/group1
```

Comment:
I think these take too long to run in CI... 25 minutes and counting. They should probably have the `unsupported` alias added and people will need to run them manually before merging redshift things.

Reply:
I can make it faster by removing the multi-node creation and delete test, that adds like 20 mins to the run.